### ISSUE TITLE
custom blocklists [suggestion] [feature]

### ISSUE THREAD

**Issue by pwn0r** [henrypp/simplewall#1046]
I'm aware this topic is both PoPuLaR and FlAmEbAit :P

simplewall uses windows spyblocker to block certain IPs and it works. However that is mainly anti-M$ block list.

there are others, and useful ones, can we add them? For instance, quite a few applications (not only on windows!) use abuse.ch feodo -- https://feodotracker.abuse.ch/blocklist/  

actual IP list link https://feodotracker.abuse.ch/downloads/ipblocklist_recommended.txt for instance. Limited (i.e. not aggressive) detected botnet addresses list, and regularly updated.  A more aggressive option -- https://feodotracker.abuse.ch/downloads/ipblocklist.txt   The 2nd option is for any such IPs detected in the last 30 days, still less than 400 entries. 

There are apps which implement such lists even on windows firewall. Automatically creating a rule and filling the scope, updating the list from online version regularly. Obviously such approach wont work with simplewall. 

Basically we would need an option to add a custom URL for IP blocklist(s). IPblock lists usually also provided (as per example above) in a similar format like windowsspyblocker, and therefore do not require any additional processing.

**Comment by thisisthelongestusernameyouhaveeverseen** [comment]
Yes, ability to add custom sources is needed, and ability to set auto-update time
and option to add the new IPs to old ones or replace them

**Comment by ltguillaume** [comment]
@henrypp Could you explain why this is closed? It's not yet possible, so why is it set to "completed"?

**Comment by henrypp** [comment]
@ltguillaume no custom blocklist will be

**Comment by ltguillaume** [comment]
Too bad. I reckon there's two things that probably would complicate such a feature:

1. The ability to parse different blocklist formats
2. With additional blocklists, you'd probably need to have a way of (temporarily) overriding a specific block rule (or permanently for a specific application) more often than currently is the case (and this is not possible at all right now, see #380 which was never answered)
